### PR TITLE
i18n add italian translation keys

### PR DESCRIPTION
**Pull request by capitanfuturo**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties [hunk 1]
```diff
@@ -4,72 +4,80 @@
...
+command.org.jkiss.dbeaver.core.txn.log.description                     = Log delle transazioni correnti (query eseguite)
+command.org.jkiss.dbeaver.core.txn.log.name                            = Log delle transazioni
+command.org.jkiss.dbeaver.core.txn.pending.description                 = Elenco di tutte le transazioni appese
+command.org.jkiss.dbeaver.core.txn.pending.name                        = Transazioni appese
+command.org.jkiss.dbeaver.ui.tools.menu.description                    = Mostra il menu contestuale con gli strumenti del database
+command.org.jkiss.dbeaver.ui.tools.menu.name                           = Menu contestuali
 
 content-type.org.jkiss.dbeaver.bookmark.name = Segnalibro
 
+keyword.org.jkiss.dbeaver.core.keyword.pref.editor.erd.label = Drigramma ERD
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties]: Diagramma maybe?

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties [hunk 2]
```diff
@@ -4,72 +4,80 @@
...
+command.org.jkiss.dbeaver.core.rollbacklabel                           = Rollback
+command.org.jkiss.dbeaver.core.transaction_mode.name                   = Modalit\u00E0 di transazione
+command.org.jkiss.dbeaver.core.txn.autocommit.description              = Modifica la modalit\u00E0 transazionale della sessione attiva
+command.org.jkiss.dbeaver.core.txn.autocommit.name                     = Auto-commit
+command.org.jkiss.dbeaver.core.txn.log.description                     = Log delle transazioni correnti (query eseguite)
+command.org.jkiss.dbeaver.core.txn.log.name                            = Log delle transazioni
+command.org.jkiss.dbeaver.core.txn.pending.description                 = Elenco di tutte le transazioni appese
+command.org.jkiss.dbeaver.core.txn.pending.name                        = Transazioni appese
+command.org.jkiss.dbeaver.ui.tools.menu.description                    = Mostra il menu contestuale con gli strumenti del database
+command.org.jkiss.dbeaver.ui.tools.menu.name                           = Menu contestuali
```
Comment (Destrolaric) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties]: Redundant tabulation

Comment (capitanfuturo) [reply plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties]: I deleted redundant tabs in my last commit. Thank you @Destrolaric for your review

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties (hunk 1 above)
Comment (capitanfuturo) [reply plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_it.properties]: Thank you @LonwoLonwo for your review, I have fixed this typo.

**Comment by LonwoLonwo** [comment]
Hello @capitanfuturo 

Thanks for your contribution!

It would be very good if you have rewritten your PR due to the extra number of commits. Unfortunately, extra commits are an obstacle to merging branches.

**Comment by capitanfuturo** [comment]
@LonwoLonwo, If it could be fine, I can make a new PR with only an atomic commit with the last changes.

**Comment by LonwoLonwo** [comment]
> I can make a new PR with only an atomic commit with the last changes.

Yes, it will be perfect, thanks!

**Pull request closed without merging**